### PR TITLE
Merge to Master PR

### PR DESCRIPTION
**Pull request by wathigo**

### CODE REVIEW

File: lib/script.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      @matrix[2][@matrix[2].index(place)] = choice
+    end
+    display
+    result = Result.new(@matrix)
+    result.check_success
+    @counter += 1
+    [result.check_success, @counter]
+  end
+
+  def display
```
Comment (AdnanAfsari) [inline lib/script.rb]: should not mix user interaction with game logic (e.g. no “puts” and “get.chomp” in lib directory!).move it to the bin.

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
**Hello @wathigo @othman-19 Nice work,almost there **

Everything looks good but you have to make a small change, please check my comment for that and make change according to that and submit for another review.

Happy Coding :)

**Comment by wathigo** [comment]
Thank you @AdnanAfsari .  We have implemented the necessary changes

**Comment by AdnanAfsari** [comment]
Hello @wathigo @othman-19 , Congratulation 👍 

Looks good, you can go ahead and merge the branch

Happy Coding

**Pull request merged**